controllers/flinkcluster_observer.go

```diff
@@ -443,6 +492,36 @@ func (observer *ClusterStateObserver) observeJobResource(
 		observedJob)
 }
 
+// observeJobPod observes job submitter pod to extract job ID and job submit log from pod termination log.
```

Comment:
Remove "to extract job ID..." because they are not part of this method.

Reply:
Updated.